src/components/include/utils/threads/thread.h

```diff
@@ -104,23 +104,46 @@ class Thread : public QObject {
 #else
 class Thread {
 #endif
+
+  typedef enum ThreadCommandEnum {
```

Comment:
@anosach-luxoft, fixed in [ce7130e](https://github.com/smartdevicelink/sdl_core/pull/707/commits/ce7130e0fa97a34d5fb143ff0764557cee82eb68)